papers/bcr-2020-010-output-desc.md

```diff
@@ -242,12 +242,14 @@ fffffffe ; child number
 			1: [ ; components
 				44, true, 0, true, 0, true ; 44'/0'/0'
 			],
-			2: 3545084735 ; origin-fingerprint
+			2: 3545084735 ; source-fingerprint
+                        3: 4 ; depth    
 		}),
 		7: 304({ ; children: crypto-keypath
 			1: [ ; components
 				1, false, [], false ; 1/*
-			]
```

Comment:
no changes are needed here

Reply:
> no changes are needed here

😅 I'm reverting that line but I don't understand why we don't need the `depth` for `children`.